server/src/controller/invitation.js

```diff
@@ -33,7 +33,14 @@ class InvitationController {
     })
       .save()
 
-    return invitation.toJSON()
+    invitation = invitation.toJSON()
+
+    const subject = 'Invitation build-raven-os-org'
+    const text = `Pour vous inscrire sur build-raven-os-org, utilisez le code suivant: ${invitation.uuid}`
```

Comment:
In English please, and the name `build-raven-os-org` is quite technical, something like `Raven's Package Builder` or anything similar would probably be more meaningful.